tests/codegen/simd/packed-simd.rs

```diff
@@ -1,4 +1,5 @@
 //@ revisions:opt3 noopt
+//@ only-x86_64
```

Comment:
This test is checking our LLVM ABI lowering as much as it is checking anything packed-simd specific, so it will be very hard to make this work uniformly across targets that use a different ABI lowering.